client/src/App.js

```diff
@@ -3,28 +3,61 @@ import Navbar from './components/Navbar.js'
 import Product from './routes/Product.js';
 import Landing from './routes/Landing.js';
 import Home from './routes/Home.js'
+import Signup from './routes/Signup'
+import Login from './routes/Login.js'
+import { useState } from 'react';
+
 
 function App() {
+  const BASE_URL = 'https://localhost:5432/grace-shopper/api';
```

Comment:
the base url is http://localhost:3001/api

Reply:
Just updated, good catch thanks